src/main/kotlin/widgets/LanguageChip.kt

```diff
@@ -0,0 +1,67 @@
+package widgets
+
+import data.Language
+import javafx.geometry.Insets
+import javafx.geometry.Pos
+import javafx.scene.input.MouseEvent
+import javafx.scene.layout.HBox
+import javafx.scene.layout.StackPane
+import javafx.scene.paint.Color
+import tornadofx.*
+
+/**
+ * A language chip is a class used to store the data of the chip for
+ * easy direct access. This will allow access to the stackpane, and elements
+ * in the stackpane without searching through the children on the parent of the
+ * stackpane for what we want to access.
+ */
+
+class LanguageChip(val language: Language,
```

Comment:
if it's a widget, you should be creating a library component- as in, it should be generic, not specific to a use case within a specific system. As in, it shouldn't be dependent on a org.wycliffeassociates.data.Language object. Think about what a chip is, and only require the necessary information to create the view. Also, styling should ideally be separated from the view.